src/QsCompiler/LanguageServer/Utils.cs

```diff
@@ -224,9 +224,9 @@ public override void Initialize(IEventSource eventSource)
 
     internal static class DotNetSdkHelper
     {
-        private static readonly Regex DotNet31Regex = new Regex(@"^3\.1\.\d+", RegexOptions.Multiline | RegexOptions.Compiled);
+        private static readonly Regex DotNet31Regex = new Regex(@"^6\.0\.\d+", RegexOptions.Multiline | RegexOptions.Compiled);
```

Comment:
Do we really need .NET 6.0 precisely or can we make this .NET 6.0 or newer? Why exactly are we tied to a precise version?

Reply:
For now, I'd like to play it safe given that during the .NET Core 3.1 days, we didn't support having only .NET 5 installations. 
Later this year, we'll need to assess the extent of our support to .NET 7, but until then, I'd like to make sure we have a well-known and supported environment.

Reply:
I think the questions is if it has to be 6.0,or can it be 6.1 or higher?

Reply:
Well, we don't know what a 6.1 version would mean in the current roadmap of .NET ... We didn't have a 5.1 for example, and currently we only have 6.0.X and 7.0.X (in preview) versions. If at some point they decide to create a .NET 6.1 SDK, we will need to understand the compatibility story and make an explicit decision on the level of support we would have. I'd rather be more constrained for now.

Reply:
@anpaz, for now it has to be 6.0.X but not higher